File: exercicios-cpp/estrutura-sequencial/exe06.cpp
#include <iostream> // cout ,cin
#include <cmath> // pow
#include <locale> // setlocale
#include <iomanip> // setprecision

# define PI 3.141592

/*
Faça um Programa que peça o raio de um círculo, calcule e mostre sua área.
*/

// area = pi * raio^2

using namespace std;

int main(){
	setlocale(LC_ALL, "portuguese");
	float raio=0, area=0;
	
	cout << "Informe o raio do circulo: "; cin >> raio;
	
	area = PI * pow(raio, 2);
	
	cout << fixed << setprecision(2);
	cout << "A area do circulo é " << area << endl;
	
	
	return 0;
}

File: exercicios-java/estrutura-sequencial/exe06.java
import java.util.Scanner;

// C:\Users\roger\OneDrive\Área de Trabalho\osvaldo\java\estrutura-sequencial

/*
Faça um Programa que peça o raio de um círculo, calcule e mostre sua área.]
*/

public class exe6{

  public static void main(String[] args){
    Scanner leitor = new Scanner(System.in);

    double raio=0, area=0;
    double pi = Math.PI; // Obtendo valor de PI

    System.out.printf("%nInforme o raio do circulo: ");
    raio = leitor.nextFloat();

    area = pi * Math.pow(raio, 2);

    System.out.printf("\nA área do circulo é %.2f.%n%n", area);

  }
}
